### ISSUE TITLE
Add a textbox to the slider in the collateral borrowing section to make entering your own ratio easier.

### ISSUE THREAD

**Issue by ivanviso** [bitshares/bitshares-ui#1475]
It is annoying in desktop. But it is definitively the most annoying when trying to use it on phone. It also looks like a relatively easy change to make.

**Comment by startailcoon** [comment]
I agree and think it would be an easy trick to just add a small input box displaying the percentage number.

I'll compose a proposal on this.

**Comment by wmbutler** [comment]
I'd like to see UX on this. I love the idea too.

**Comment by startailcoon** [comment]
This is my idea of how to apply this.
The input should be set either on the left or right of the slider. The value below the slider could be removed.

The user should be able to input a number with a precission of 2 decimals.

![bitshares-cr_input](https://user-images.githubusercontent.com/12114550/39652698-bb769588-4fee-11e8-8b9a-6406d675c379.gif)
